modules/pytket-aqt/pytket/extensions/aqt/backends/aqt.py

```diff
@@ -176,6 +179,9 @@ def default_compilation_pass(self, optimisation_level: int = 1) -> BasePass:
                     _aqt_rebase(),
                     RemoveRedundancies(),
                     EulerAngleReduction(OpType.Ry, OpType.Rx),
+                    SimplifyInitial(
```

Comment:
`SimplifyInitial` includes `RemoveRedundancies` by defualt, but it might introduce an `Rx` gate at the start, which could potentially be reduced by `EulerAngleReduction`. So I think replacing `RemoveRedundancies` with `SimplifyInitial` would be the best approach.